Misc/NEWS.d/next/Windows/2018-02-07-17-50-48.bpo-29248.Xzwj-6.rst

```diff
@@ -0,0 +1 @@
+fix os.readlink() on Windows

```

Comment:
Please make the news entry more descriptive and add "Patch by Craig Holmquist and Your Name."

Reply:
done